src/server.js

```diff
@@ -1,40 +1,42 @@
-import express from 'express';
-import bodyParser from 'body-parser';
+import express from "express";
+import bodyParser from "body-parser";
 
-import { log } from './utils/logger';
+import { log } from "./utils/logger";
 
 const app = express();
 
 // Connect basic middleware
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({
-  extended: true,
-}));
+app.use(
+  bodyParser.urlencoded({
+    extended: true
+  })
+);
 
 // Hook up our routes
-app.use('/', require('./routes'));
+app.use("/", require("./routes"));
 
 // Start the server listening
 const server = app.listen(3000, () => {
-  log.info('Demonstration Express API Running');
+  log.info("Demonstration Express API Running");
 });
 
 // Handle Errors
 app.use((err, req, res) => {
   // Log the request with error noted
-  log.error('ERROR - %s, %s', req.method, req.url);
+  log.error("ERROR - %s, %s", req.method, req.url);
 
   // Log the Error
   log.error(err.stack);
 
   // Return 500 error code
-  res.status(500).send({ message: 'Error' });
+  res.status(500).send({ message: "Error" });
 });
 
 // Handle 404s
-app.use((req, res, next) => {
+app.use((req, res, res, next) => {
   res.status(404).send({
-    message: 'Houston, we\'ve had a problem here',
-    urlRequested: `${req.method} ${req.url}`,
+    message: "This is not a valid page. Please check your url. ",
+    urlRequested: `${req.method} ${req.url}`
   });
```

Comment:
just invoke the next parameter 
app.use((req, res, res, next) => {
  res.status(404).send({	  res.status(404).send({
    message: 'Houston, we\'ve had a problem here',	    message: "This is not a valid page. Please check your url. ",
    urlRequested: `${req.method} ${req.url}`,	    urlRequested: `${req.method} ${req.url}`
  });
   move to the next middleware  if a middleware is applied to the handle function.
  next()
  });